### DISCUSSION TITLE
More integrations 🖖🏼

### DISCUSSION THREAD

**Discussion by ricardogouveia3** [dailydotdev/daily#873] [Feature Requests]
Hi y'all.

It's very cool that daily.dev (I'm primarily using the new tab ext) encourages us to read by a gamification process. But it would be cool to allow us to have more control about our reading goals and achievements. 

I'm not aware about the technical viability but it would be cool to have integrations to:

- Chrome reading list: I'm really not sure if this can be accessed by an extension; it pretty new, but it seems to be the Chrome way to kill the reading list extensions, and totally makes sense from an user perspective to keep track of readings on just one source

- Notion: A lot of people are using notion to organize their life, and theres a lot of templates for reading lists and trackers for download. Even thought people are using this templates mainly to keep tracking books they read, I think it would be cool to keep track of technical reading as well, both for things to read later and for thing already read, but useful for future reference.

**Comment by idoshamun** [comment]
Hi @ricardogouveia3,

This is a great idea that is currently out of our scope.
We hope to release an API in the future that will allow devs to build such integrations 🤩
